test/bootstrap.sh

```diff
@@ -147,14 +161,43 @@ cleos transfer eosio eptestuserse "1000 EOS"
 cleos transfer eosio eptestusersf "1000 EOS"
 cleos transfer eosio eptestusersg "1000 EOS"
 
-## Deploy contracts
-echo -e "${CYAN}-----------------------DEPLOYING EVERIPEDIA CONTRACTS-----------------------${NC}"
-cleos set contract everipediaiq ../everipediaiq/
-cleos set contract eparticlectr ../eparticlectr/
-cleos set account permission everipediaiq active '{ "threshold": 1, "keys": [{ "key": "EOS6XeRbyHP1wkfEvFeHJNccr4NA9QhnAr6cU21Kaar32Y5aHM5FP", "weight": 1 }], "accounts": [{ "permission": { "actor":"eparticlectr","permission":"eosio.code" }, "weight":1 }, { "permission": { "actor":"everipediaiq","permission":"eosio.code" }, "weight":1 }] }' owner -p everipediaiq
-cleos set account permission eparticlectr active '{ "threshold": 1, "keys": [{ "key": "EOS6XeRbyHP1wkfEvFeHJNccr4NA9QhnAr6cU21Kaar32Y5aHM5FP", "weight": 1 }], "accounts": [{ "permission": { "actor":"eparticlectr","permission":"eosio.code" }, "weight":1 }, { "permission": { "actor":"everipediaiq","permission":"eosio.code" }, "weight":1 }] }' owner -p eparticlectr
-
-# Create and issue token
-echo -e "${CYAN}-----------------------CREATING IQ TOKEN-----------------------${NC}"
-cleos push action everipediaiq create "[\"everipediaiq\", \"100000000000.000 IQ\"]" -p everipediaiq@active
-cleos push action everipediaiq issue "[\"everipediaiq\", \"10000000000.000 IQ\", \"initial supply\"]" -p everipediaiq@active
+if [ $REBUILD_EVERIPEDIA_CONTRACTS -eq 1 ]; then
+    cd ..
+
+    echo "Deleting old everipediaiq files..."
+    cd everipediaiq
```

Comment:
To run one command per dir, you can use the following one-line syntax (instead of 3 lines):
```
(cd everipediaiq && rm -f everipediaiq.abi everipediaiq.wasm everipediaiq.wast)
```